lib/plugins/attributeAlias.js

```diff
@@ -27,10 +27,6 @@
 
 var config = require('../commonConfig'),
     utils = require('./pluginUtils'),
-    logger = require('logops'),
```

Comment:
Not sure if remove this is a good idea... I understand the reason is hat *currently* is an unused variable but I think is a good idea to have the logger object initiated (in partciular, the op string associated to each module) in the case logger needs to be used *in the future*.

Could a `/* jshin ... */` expection be used for this?

Reply:
Fixed with 749e1751e41ea144efdd10e6ef2f45a4f73a00bd and f467b5aaa51fdb4391d84664ff2129d92599dcba